subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dependencies/AbstractExternalModuleDependency.java

```diff
@@ -90,22 +88,7 @@ public String getVersion() {
 
     @Override
     public boolean isForce() {
-        return force;
-    }
-
-    @Override
-    @SuppressWarnings("deprecation")
-    public ExternalModuleDependency setForce(boolean force) {
-        validateMutation(this.force, force);
```

Comment:
Should we still validate this for `setForceForEnforcedPlatform`?

Reply:
Seems likely, good catch.